src/main/java/com/targomo/client/api/TravelOptions.java

```diff
@@ -124,24 +124,28 @@
     @Transient private Integer multiGraphSerializationMaxGeometryCount                       = null;
     @Transient private MultiGraphAggregationType multiGraphAggregationType                   = null;
     @Transient private Boolean multiGraphAggregationIgnoreOutlier                            = null;
-    @Transient private Integer multiGraphAggregationOutlierPenalty                           = null;
+    @Transient private Float multiGraphAggregationOutlierPenalty                             = null;
     @Transient private Double multiGraphAggregationMinSourcesRatio                           = null;
     @Transient private Integer multiGraphAggregationMinSourcesCount                          = null;
     @Transient private Double multiGraphAggregationMaxResultValueRatio                       = null;
-    @Transient private Integer multiGraphAggregationMaxResultValue                           = null;
+    @Transient private Float multiGraphAggregationMaxResultValue                             = null;
     @Transient private Set<String> multiGraphAggregationFilterValuesForSourceOrigins         = null;
-    @Transient private Map<String, SourceParameter> multiGraphSourceParameters               = null;
-    @Transient private Map<String, AggregationConfig> multiGraphPreAggregationPipeline       = null;
+    @Transient private Double multiGraphAggregationGravitationExponent                       = null;
+    @Transient private LinkedHashMap<String, AggregationInputParameters> multiGraphAggregationInputParameters = null;
```

Comment:
I think we decided to make the inputParameters not ordered, i.e. Map instead of LinkedHashMap

Reply:
This ordering came from Gent's branch, which needs the aggregation pipeline in order